src/mainPage.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Now that the main page elements have been added to App.tsx, I imagine we won't need this standalone html file. If you added it to show the work you did along the way, it'll still be there in your series of commits and in this review, to prove you did write it, even if it doesn't make it into the main branch. If we wanted to keep it though as our prototype, we could move it into its own `prototype` directory apart from `src`.

Reply:
I'm open to other team members' thoughts too if anyone wants to chime in.

Reply:
Yeah I knew it wouldn't go into the main branch, I realized it was incorrect after I created the pull request. I just didn't know how to go about removing it without completely deleting the work I did / time I put in.